src/hotspot/share/code/codeBlob.hpp

```diff
@@ -506,17 +403,14 @@ class RuntimeStub: public RuntimeBlob {
 
   static void free(RuntimeStub* stub) { RuntimeBlob::free(stub); }
 
-  // Typing
-  bool is_runtime_stub() const                   { return true; }
-
   address entry_point() const                    { return code_begin(); }
 
   // GC/Verification support
-  void preserve_callee_argument_oops(frame fr, const RegisterMap *reg_map, OopClosure* f)  { /* nothing to do */ }
+  virtual void preserve_callee_argument_oops(frame fr, const RegisterMap *reg_map, OopClosure* f) override { /* nothing to do */ }
```

Comment:
`override` is not required by C++. You do however mark all virtual methods with `override` if any of the functions are marked with `override`. I think it would be good to have a HotSpot code style discussion about this (but not in this RFE).

Reply:
I put `virtual/override` cleanup in CodeBlob as additional suggestion in followup RFE JDK-8329628.